LATEX.md

```diff
@@ -50,3 +50,84 @@ literally display a dollar sign you can use `\$`.
     it's exactly three seconds before i honk your $\int -xe^{x^2} dx$ nose and
     pull your underwear over your head ...
     ````
+
+## Macros
```

Comment:
I think its a little lengthy. Would would be great is to distil it down to three main things
- A sentence telling the user that MPP supports persistent macro definitions.
- What file the definitions need to be placed in.
- How to make those definitions in the file (ie the syntax).